### PR TITLE
Test new api

### PR DESCRIPTION
**Pull request by ShaneKilkelly**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -4,7 +4,7 @@ defmodule HammerTestbed.Mixfile do
   def project do
     [app: :hammer_testbed,
      version: "0.0.1",
-     elixir: "~> 1.4",
+     elixir: "~> 1.5",
```
Comment (sobolevn) [inline mix.exs]: Any reason to bump the minimal version?

File: lib/hammer_testbed/application.ex [hunk 2]
```diff
@@ -10,7 +10,6 @@ defmodule HammerTestbed.Application do
     children = [
       # Start the endpoint when the application starts
       supervisor(HammerTestbed.Web.Endpoint, []),
```
Comment (sobolevn) [inline lib/hammer_testbed/application.ex]: Trailing coma.

File: mix.exs (hunk 1 above)
Comment (ShaneKilkelly) [reply mix.exs]: I'm actually not sure how that happened, I must have been testing something

**Comment by ShaneKilkelly** [comment]
@sobolevn, updated :)

**Pull request merged**